src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/resourceSubscriptions.jsx

```diff
@@ -18,17 +18,27 @@ export default class Subscriptions extends React.Component {
     permissions: PropTypes.object.isRequired,
     events: PropTypes.array.isRequired,
     onChange: PropTypes.func.isRequired,
+    webhookDisabled: PropTypes.bool.isRequired,
+  };
+
+  static defaultProps = {
+    webhookDisabled: false,
   };
 
   constructor(...args) {
     super(...args);
-    this.state = {events: this.props.events};
```

Comment:
We have `events` as a prop which is updated with the `onChange` callback so the state field is unnecessary.